ament_lint_cmake/ament_lint_cmake/main.py

```diff
@@ -70,6 +73,8 @@ def main(argv=sys.argv[1:]):
     # invoke cmake lint
     cmakelint._lint_state.config = cmakelint._DEFAULT_CMAKELINTRC
     cmakelint._lint_state.SetFilters(args.filters)
+    if args.linelength is not None:
+        cmakelint._lint_state.SetLineLength(str(args.linelength))
```

Comment:
Kind of strange that we're casting from `int` to `str` and then back to `int`, but it should be fine.